protocol/communities/community.go

```diff
@@ -1408,17 +1408,17 @@ func (o *Community) MarshaledDescription() ([]byte, error) {
 	return o.marshaledDescription()
 }
 
-func (o *Community) toBytes() ([]byte, error) {
+func (o *Community) toProtocolMessageBytes() ([]byte, error) {
 
 	// This should not happen, as we can only serialize on our side if we
 	// created the community
-	if o.config.PrivateKey == nil && len(o.config.MarshaledCommunityDescription) == 0 {
+	if o.config.PrivateKey == nil && len(o.config.CommunityDescriptionProtocolMessage) == 0 {
 		return nil, ErrNotAdmin
 	}
 
 	//We are not owner or admin, use the received serialized version
 	if !o.IsOwnerOrAdmin() {
```

Comment:
This is a bug in current master, only owner should be able to marshal and sign the description. /cc: @mprakhov @cammellos